src/lib/plugin.js

```diff
@@ -379,8 +379,8 @@ class FiveBellsLedger extends EventEmitter2 {
     if (!res || res.statusCode !== 200) { throwErr() }
 
     // note that the fivebells ledger API uses 'scale' instead of 'currency_scale'.
-    // the 'ilp_prefix' field is not mandatory
-    if (!res.body.currency_code || !res.body.scale) { throwErr() }
+    // fields like 'ilp_prefix' and 'currency_code' are not mandatory
+    if (!res.body.connectors || !res.body.scale) { throwErr() }
```

Comment:
@michielbdejong on the branch of the ILP kit I am working on all integration tests [pass](https://circleci.com/gh/interledgerjs/ilp-kit/1646). Maybe this version of the plugin is not yet used by ILP kit?